### ISSUE TITLE
Announcing invidious.io, and the deprecation of invidio.us

### ISSUE THREAD

**Issue by TheFrenchGhosty** [iv-org/invidious#1694]
TL;DR at the bottom

---

Due to multiple issue with invidio.us:

- Domain still owned by Omarroth

- .us domains requiring US citizenship

- The content *was* hosted on a personal server used by omarroth for other things (this is now changed)

---

Changes to invidio.us:

invidio.us is now deprecated, and *later* will only redirect to invidious.io.

The content still on invidio.us is now hosted on a VPS, dedicated to Invidious, which @Perflyst and me ( @TheFrenchGhosty ) control.

---

News:

A new domain was obtained: invidious.io and the content is now hosted on an Hetzner VPS in Germany.

Both the domain, the VPS, and everything else related to Invidious is now managed by @Perflyst and me ( @TheFrenchGhosty ), the project is now *bus factor* resistant.

The invidio.us domain is still owned by Omarroth (this won't ever change).

We would like to deeply thanks everyone who donated cryptocurrencies, allowing us to buy the new domain, and pay for the VPS for some months. A detail of the January expenses will soon be available in https://github.com/iv-org/transparency

---

**TL;DR:**

invidio.us is now deprecated, invidious.io is the new domain.

The invidious-redirect website ( invidio.us ) is now  https://redirect.invidious.io/

The instance API ( instances.invidio.us ) is now https://api.invidious.io/

The uptimerobot page ( uptime.invidio.us ) is now https://uptime.invidious.io/

https://git.invidious.io/ redirects to the Github page of the project: https://github.com/iv-org/invidious/ 

https://docs.invidious.io/ redirects to the documentation website of the project: https://github.com/iv-org/documentation/

https://invidious.io/ will become in the future a "project presentation" page (like https://searx.me/ ), we will use a markdown framework like [Hugo](https://github.com/gohugoio/hugo).

Projects using *something* from invidio.us need to move to the new domain.

---

[FreeTube](https://github.com/FreeTubeApp/FreeTube) by @PrestonN, and [NewPipe](https://github.com/TeamNewPipe/NewPipe) by @TeamNewPipe already moved, other developers are **strongly** encouraged to move as soon as possible.

Some developers that have projects that currently depend on invidio.us:

- @SimonBrazell (with [Privacy Redirect](https://github.com/SimonBrazell/privacy-redirect))

- @zedeus (with [Nitter](https://github.com/zedeus/nitter))

- @cloudrac3r (with [Bibliogram](https://sr.ht/~cadence/bibliogram/))

- tom79 (with [UntrackMe](https://framagit.org/tom79/nitterizeme)) - Edit: The developers [were contacted on Mastodon](https://toot.fedilab.app/@apps/105601094336953893) UntrackMe isn't affected because the instances list is self-contained: https://framagit.org/tom79/fedilab_app/-/blob/master/content/untrackme_instances/payload_2.json

I strongly enourage everyone to ping developers of projects that you know use or depend on something hosted on Invidio.us.

**Comment by SimonBrazell** [comment]
So no "official instance" like invidio.us used to be?

**Comment by TheFrenchGhosty** [comment]
@SimonBrazell invidio.us was shut down months ago, and was just a redirect page.

See #1321

This is just a move to a new domain name.

**Comment by SimonBrazell** [comment]
Yes I realise that but wasn't that just because you guys couldn't access the domain / server? I was just thinking the project would benefit from having an officially maintained instance again like prior to the author's departure.

**Comment by SimonBrazell** [comment]
And thanks for the heads up, much appreciated, but I don't think this change will impact Privacy Redirect.

**Comment by TheFrenchGhosty** [comment]
@SimonBrazell 

>I was just thinking the project would benefit from having an officially maintained instance again like prior to the author's departure.

I would like this to be a thing, but we don't have the funding for this. I did the math a while ago, and concluded that to have a public instance (with dash enabled by default) we basically need to spend more than 300€ monthly.

This isn't possible for now, and won't be for a really long time.

**Comment by SimonBrazell** [comment]
> we basically need to spend more than 300€ monthly. This isn't possible for now, and won't be for a really long time.

Fair enough, those costs sound very high to me but I guess the servers I build don't typically deal with the kind of load invidious would.

Thanks again for letting me know about this and clearing this up, all the best!

**Comment by TheFrenchGhosty** [comment]
@Mennaruuk Thanks a lot for contacting all those projects developers!

**Comment by Mennaruuk** [comment]
ABSOLUTELY! My pleasure :)

**Comment by probonopd** [comment]
Will there be something like https://instances.invidious.io/ anytime soon?
A page where one can link to, which contains the different instances (without mentioning "shut down", "coming soon", etc.)?

**Comment by TheFrenchGhosty** [comment]
@probonopd Yes this is planed. For now you can use https://docs.invidious.io/Invidious-Instances.md

**Comment by probonopd** [comment]
Is "uses Cloudflare" a good or a bad thing?

**Comment by TheFrenchGhosty** [comment]
@probonopd it's bad for privacy. Cloudflare is US based, has a monopoly, and is a PRISM member.

**Comment by probonopd** [comment]
Then maybe this should be stated very clearly, especially the PRISM part.

**Comment by TheFrenchGhosty** [comment]
@probonopd You're right, I remembered wrong, they aren't a PRISM member (officially).

However, this is still a thing: https://codeberg.org/themusicgod1/cloudflare-tor -  http://www.crimeflare.org:82/

**Comment by Atemu** [comment]
Sorry but what you just posted looks like a load of near-baseless FUD. (or a bad joke, actually not sure which)

CF's large "market share" is certainly worrying and a publicly owned company cannot be trusted to do anything but act in the interest of their investors of course but so far they haven't shown any truly malicious intent.

If you care about privacy, you shouldn't be sending private/personally identifying information to anyone in the first place, no matter the in-transit encrytion.  
Luckily, Invidious keeps such information to a bare minimum, so none of that really matters in this case. I'd almost feel comfortable communicating with it in plaintext honestly.  
TLA organisations conspiring with an evil internet monopolist just to find out which cat videos you watch on an online youtube proxy is simply not something that needs to be in anyone's threat model, even if it was real.

Having an Invidious instance behind CF is undoubtedly advantageous and the disadvantages are theoretical at best, if they exist at all.  
Such instances should not be treated any differently because of some FUD two(?) people threw on the internet.  
It would still be useful to know whether an instance is behind CF though but mainly because of the scalability and uptime improvements that usually entails.

**Comment by hackerncoder** [comment]
There is more than just FUD. CloudFlare is very much against Tor, and the reason people might be using Invidious in the first place could very well be because YouTube blocks Tor too. Therefore telling people which instances to avoid is a good idea (IMO)

**Comment by Atemu** [comment]
> CloudFlare is very much against Tor

What do you mean by that?

Do they *actively* try to suppress and/or hurt the TOR network itself (as you imply) or do they just block exit nodes?

Those are very different things because, believe it or not, people who know they can't be identified are sometimes inclined to abuse public resources such as TOR exit nodes for actually nefarious and malicious purposes.  
CF wanting to protect their customers against such threats is more than understandable and has nothing to do with maliciousness or being "against TOR".  
The same would happen to any public proxy, regardless of whether it is reachable via TOR or not.

> the reason people might be using Invidious in the first place could very well be because YouTube blocks Tor too. Therefore we should tell people which instances to avoid (IMO)

I don't think it's a good idea to burden the TOR network with large amounts of data such as video (pseudonymising clear-net proxies (i.e. "VPN"s) should be preferred here) but that would indeed be a use-case where being behind CF may be adverse.

**Comment by Testsr** [comment]
>  CloudFlare is very much against Tor
Why, then are they giving official help support for Tor then? 
https://support.cloudflare.com/hc/en-us/articles/203306930-Understanding-Cloudflare-Tor-support-and-Onion-Routing
And why do they make blog posts about how they are working to get TOR working? 
https://blog.cloudflare.com/the-trouble-with-tor/
If they really were against TOR, they would not do these things.

**Comment by Hahlh** [comment]
Hey everyone! 
I really appreciate, that you are caring about privacy and constructive discussions like these help us move forward and figure stuff out as a community.

Having said this, would it be alright for you to have this moved to the discussions instead?
I have subscribed to this so that I would be updated about the further developments regarding invidious.io and I believe, that a separate issue or discussion would be preferrable to keep this issue on topic.

If you and the maintainer prefer to keep this here, no worries.

A wonderful day to everybody!

**Comment by hackerncoder** [comment]
> Do they actively try to suppress and/or hurt the TOR network itself (as you imply) or do they just block exit nodes?

They block exit nodes. I was speaking from feelings, that was wrong.

> inclined to abuse public resources such as TOR exit nodes for actually nefarious and malicious purposes.

Yes.

> I don't think it's a good idea to burden the TOR network with large amounts of data such as video 

I don't think it's a problem. The Tor Network has [a lot of unused bandwidth](https://metrics.torproject.org/bandwidth.html). Tor also needs cover traffic, day-to-day and other random traffic to keep others anonymous.

> but that would indeed be a use-case where being behind CF may be adverse.

:+1: 

&nbsp;

> If they really were against Tor, they would not do these things.

I was speaking from feelings, that was wrong.

> And why do they make blog posts about how they are working to get TOR working?
> https://blog.cloudflare.com/the-trouble-with-tor/

That's good. Also: https://blog.torproject.org/trouble-cloudflare.

<pedantic> [It's Tor](https://support.torproject.org/about/why-is-it-called-tor/). Not [TOR](https://matt.traudt.xyz/posts/tor_spelling/) </pedantic>

**Comment by grahamperrin** [comment]
https://github.com/iv-org/invidious/issues/1694#issuecomment-785632864

> … to the discussions …

👍

▶ https://github.com/iv-org/invidious#contact ▶ [Matrix room](https://matrix.to/#/#invidious:matrix.org), or `#invidious` on freenode.

**Comment by TheFrenchGhosty** [comment]
I definitely agree with @Hahlh 

Cloudflare is fundamentally bad, and the way we handle it won't change (we might actually write more reasons why it should be avoided).

Please move this Cloudflare discussion to another issue.

**Comment by drogga** [comment]
...I was really hoping for support/to be compatible with older network protocols too this time around (in the new one), like TLS 1.0, 1.1 & 1.2, SSL's and/or whichever of this type/kind, so I can actually open/load this from/with some of my 'bricks' and pick a instance that will work/load for me...

BTW - I can use only some of the instances/mirrors on those devices (with older browsers - on purpose or not, for couple of reasons), the ones that don't enforce TLS 1.3 and the .Onion(s) by taking advantage of Tor2Web (.ly), but most of them don't function properly (they use old source vers. or whatever) and since cookies aren't remembered/stored via this method - I can't make any changes/set settings preferences, or even switch to the dark theme...

Also it would be nice if the default rez. (JS-off, Android's native ExoPlayer - rez. user preferences/settings are not being used in this case at all) can be brought/reverted back to 720p, like before (long time ago, over a year), instead of 360p as it's currently - on code basis, so all the instances can use it and don't change it.

**Comment by unixfox** [comment]
> ...I was really hoping for support/to be compatible with older network protocols too this time around (in the new one), like TLS 1.0, 1.1 & 1.2, SSL's and/or whichever of this type/kind, so I can actually open/load this from/with some of my 'bricks' and pick a instance that will work/load for me...
> 
> BTW - I can use only some of the instances/mirrors on those devices (with older browsers - on purpose or not, for couple of reasons), the ones that don't enforce TLS 1.3 and the .Onion(s) by taking advantage of Tor2Web (.ly), but most of them don't function properly (they use old source vers. or whatever) and since cookies aren't remembered/stored via this method - I can't make any changes/set settings preferences, or even switch to the dark theme...
> 
> Also it would be nice if the default rez. (JS-off, Android's native ExoPlayer - rez. user preferences/settings are not being used in this case at all) can be brought/reverted back to 720p, like before (long time ago, over a year), instead of 360p as it's currently - on code basis, so all the instances can use it and don't change it.

Feel free to run your Invidious instance if you want to use it on older devices. We are not going to support TLS 1.0 & TLS 1.1 anymore because those TLS protocols are bad for the security.

**Comment by drogga** [comment]
...I was really hoping for support/to be compatible with older network protocols too this time around (in the new one), like TLS 1.0, 1.1 & 1.2, SSL's and/or whichever of this type/kind, so I can actually open/load this (.io redirect site) from/with some of my 'bricks' and pick a instance that will work/load for me...
Please don't just don't tell or try to convince me that v1.3 is enforced basically on everything nowadays for a security reasons, privacy (which is simply a joke in 'modern' days) or whatever else bullshit - there's no point.

BTW - I can use only some of the instances/mirrors on those devices (with older browsers - on purpose or not, for couple of reasons), the ones that don't enforce TLS 1.3 and the .Onion(s) by taking advantage of Tor2Web (.ly), but most of them don't function properly (they use old source vers. or whatever) and since cookies aren't remembered/stored via this method - I can't make any changes/set settings preferences, or even switch to the dark theme...

Also it would be nice if the default rez. (JS-off, Android's native ExoPlayer - rez. user preferences/settings are not being used in this case at all) can be brought/reverted back to 720p, like before (long time ago, over a year), instead of 360p as it's currently - on code basis, so all the instances can use it and don't change it.
The DRM (like music videos) content (HLS/Dash - separate A/V streams) can still be made to keep playing in 360p (MP4's with mux-ed/build-in audio) in those cases.

Also: Other than YT (App or Site) nothing can open age restricted channels like "Bud Light", for which you need to login to YT and be over 18\21 to even see the channel's content (uploads\videos), even none of the invidious instances can't open\load those channels, like the one i mentioned, let along NewPipe, SkyTube ot whatever else, tho their content is playable as a separate individual videos...

**Comment by TheFrenchGhosty** [comment]
@drogga This is offtopic, and @unixfox answered you. We're not going to support outdated protocols.

**Comment by drogga** [comment]
Ok.
Also there is more than that, my comment contains multiple suggestion (let's say), ignore it if you want, I did/had wrote what I wanted to say.
No, it's definitely not a off-topic !, if you reRead some of the other (not hidden) comments, then you will see off-topic ones.

**Comment by unixfox** [comment]
@drogga

If you want to suggest some modifications into the code of Invidious then please open a separate issue. This issue is about the website invidious.io not about restricted channels nor the default resolution of a video.

Let's keep this issue clean and exempt of off-topic. Multiple people are subscribed to this issue and receive a notification for every new comment. The least that they want it's to only have comments related to the initial subject into their notification inbox.

Thank you for your understanding.

**Comment by TheFrenchGhosty** [comment]
@drogga 

>No, it's definitely not a off-topic !, if you reRead some of the other (not hidden) comments, then you will see off-topic ones.

So asking about instance-stuff in an annoucement of the new domain of the project isn't offtopic now?

Just open your own issues.

**Comment by HkopMD** [comment]
It would be cool if we could donate on a [Patreon](https://www.patreon.com) [opencollective](https://opencollective.com) [liberapay](https://liberapay.com) or [Ko-fi](https://ko-fi.com)

**Comment by TheFrenchGhosty** [comment]
@HkopMD We already explained before that it's not really possible for us to do that.

However if you really want to donate you can donate to people who work on invidious directly, most of them have way to receive donations.

**Comment by HkopMD** [comment]
> @HkopMD We already explained before that it's not really possible for us to do that.
> 
> Also, snce, currently donations are owned by the _project_ and not by _someone_, it's even less do-able.
> 
> However if you really want to donate you can donate to people who work on invidious directly, most of them have way to receive donations, but yes, those donations won't go toward the project.

:( I wanted to donate so you guys have the money to make an official invidious instance

**Comment by unixfox** [comment]
> > @HkopMD We already explained before that it's not really possible for us to do that.
> > Also, snce, currently donations are owned by the _project_ and not by _someone_, it's even less do-able.
> > However if you really want to donate you can donate to people who work on invidious directly, most of them have way to receive donations, but yes, those donations won't go toward the project.
> 
> :( I wanted to donate so you guys have the money to make an official invidious instance

Unfortunately for the time being running an official invidious instance would be too expensive. The maintainers of Invidious already talked about it, and they decided against deploying it one.

Maybe they will reconsider this possibility when Invidious will be rewritten (in another programming language).

As a maintainer of yewtu.be I would surely offer my help if one day they want to deploy a production ready official Invidious instance.
